src/extension.ts

```diff
@@ -20,7 +20,8 @@ import { DefaultAWSStatusBar } from './shared/defaultStatusBar'
 import { EnvironmentVariables } from './shared/environmentVariables'
 import { ext } from './shared/extensionGlobals'
 import { safeGet, showQuickStartWebview, toastNewUser } from './shared/extensionUtilities'
-import * as logFactory from './shared/logger'
+import { getLogger } from './shared/logger'
+import { initialize as activateLogger } from './shared/logger/logger'
```

Comment:
Why not just change the function name to activateLogger?

Reply:
Can we think of something better for the file path than logger/logger?

Reply:
This parallels how we [activate the SAM functionality](https://github.com/aws/aws-toolkit-vscode/blob/master/src/extension.ts#L25). In an upcoming separate change, `initialize` will be moved to a file called 'activation' and be renamed to `activate`, as a convention for extension activation calls.

Also in an upcoming change, `logger.ts` will primarily contain the `Logger` interface, and not much else. We won't see much of `.../logger/logger`